### PR TITLE
Some balance changes to attachments

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Buffs/Nerfs attachments

## Why It's Good For The Game

It makes them less poopy

## Changelog
:cl:
Balance: Some value changes to marine attachments
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -321,12 +320,12 @@ Defined in conflicts.dm of the #defines folder.
 	attach_delay = 10 //Bayonets attach/detach quickly.
 	detach_delay = 10
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
-	melee_mod = 20 //35 for a rifle, comparable to 37 before. 40 with the stock, comparable to 42.
+	melee_mod = 25 //35 for a rifle, comparable to 37 before. 40 with the stock, comparable to 42.
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]:
```suggestion
	melee_mod = 25
```

File: code/modules/projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -348,10 +347,9 @@ Defined in conflicts.dm of the #defines folder.
 	slot = "muzzle"
 	icon_state = "ebarrel"
 	attach_icon = "ebarrel_a"
-	attach_shell_speed_mod = -1
-	accuracy_mod = 0.20
-	accuracy_unwielded_mod = 0.15
-	damage_mod = -0.05
+	attach_shell_speed_mod = 1
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]: This should come with a description change.

File: code/modules/projectiles/gun_attachables.dm [hunk 3]
```diff
@@ -362,12 +360,11 @@ Defined in conflicts.dm of the #defines folder.
 	slot = "muzzle"
 	icon_state = "hbarrel"
 	attach_icon = "hbarrel_a"
-	attach_shell_speed_mod = 1
 	accuracy_mod = -0.30
 	damage_mod = 0.30
 	delay_mod = 3
 	scatter_mod = 5
-	accuracy_unwielded_mod = -0.40
+	accuracy_unwielded_mod = -0.30
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]:
```suggestion
	accuracy_unwielded_mod = -0.3
```

**Comment by psykzz** [comment]
@Kuro-ai (im terrible at marine balance) have you had time to thoroughly test this change and happy with the results?

**Comment by Kuro-ai** [comment]
I only played one match with the changes but they seem good to me

**Comment by ma44** [comment]
If the changelog had a better description for all the changes done that would be excellent

**Comment by Rohesie** [comment]
UnTMed until the change request is done.

File: code/modules/projectiles/gun_attachables.dm [hunk 4]
```diff
@@ -578,12 +573,12 @@ Defined in conflicts.dm of the #defines folder.
 	slot = "rail"
 	icon_state = "autoloader"
 	attach_icon = "autoloader_a"
-	accuracy_mod = -0.20
+	accuracy_mod = -0.15
 	scatter_mod = 5
-	delay_mod = -1
+	delay_mod = -1.25
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]:
```suggestion
	delay_mod = -0.125 SECONDS
```

File: code/modules/projectiles/gun_attachables.dm [hunk 5]
```diff
@@ -601,12 +596,12 @@ Defined in conflicts.dm of the #defines folder.
 	attach_icon = "sniperscope_a"
 	desc = "A rail mounted zoom sight scope. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
 	slot = "rail"
-	aim_speed_mod = 1 //Extra slowdown when aiming
-	wield_delay_mod = 0.6 SECONDS
+	aim_speed_mod = 0.6 //Extra slowdown when aiming
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]:
```suggestion
	aim_speed_mod = 0.06 SECONDS //Extra slowdown when aiming
```

File: code/modules/projectiles/gun_attachables.dm [hunk 6]
```diff
@@ -638,12 +633,14 @@ Defined in conflicts.dm of the #defines folder.
 	attach_icon = "miniscope_a"
 	desc = "A small rail mounted zoom sight scope. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
 	slot = "rail"
-	wield_delay_mod = 0.4 SECONDS
+	wield_delay_mod = 0.2 SECONDS
+	accuracy_mod = 0.05
+	accuracy_unwielded_mod = -0.05
+	aim_speed_mod = 0.4
```
Comment (Rohesie) [inline code/modules/projectiles/gun_attachables.dm]:
```suggestion
	aim_speed_mod = 0.04 SECONDS
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-7). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**